lib.js

```
@@ -0,0 +1,12 @@
+'use strict'
+
+const _ = require('lodash')
+
+const getErr = err => _.isString(err) ? new Error(err) : err
+
+const buildErr = err => err ? new Error(err.message || err) : null
```

isn't this hiding the original error trace?